streams/src/main/java/org/apache/kafka/streams/state/internals/TimestampedKeyValueStoreBuilder.java

```diff
@@ -91,11 +91,13 @@ private KeyValueStore<Bytes, byte[]> maybeWrapLogging(final KeyValueStore<Bytes,
     }
 
     private final static class InMemoryTimestampedKeyValueStoreMarker
+        extends WrappedStateStore<KeyValueStore<Bytes, byte[]>, Bytes, byte[]>
```

Comment:
was this related to the changes in this PR? if not, let's leave it out for now, especially as we'd probably want to clean it up a bit further and change the `wrapped` invocations to `super` throughout the class

but feel free to do a followup PR with this!

Reply:
yes it was, we needed it in order to pass the tests that I added which check that `InMemoryXStore` is in the store heirarchy - otherwise it would stop at this change

Reply:
ah, ok, that makes sense. In that case yeah, can you just make a quick pass and update the calls in this? thanks!